config/locales/oc.yml

```diff
@@ -0,0 +1,5 @@
+en:
```

Comment:
It should be `oc` here, can you pls fix that :)

Reply:
oupsss, fixed!